app/pandas.ipynb

```diff
@@ -51,24 +52,90 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
    "source": [
     "data = {\n",
     "    \"Name\": [\"John\", \"Emily\", \"Ryan\"],\n",
     "    \"Age\": [16, 28, 22],\n",
     "    \"City\": [\"New York\", \"Los Angeles\", \"Chicago\"],\n",
     "}\n",
     "\n",
-    "# write your code here"
+    "df = pd.DataFrame(\n",
+    "    {\"Name\": [\"John\", \"Emily\", \"Ryan\"],\n",
+    "    \"Age\": [16, 28, 22],\n",
+    "    \"City\": [\"New York\", \"Los Angeles\", \"Chicago\"]}\n",
+    ")\n",
+    "df"
```

Comment:
The creation of the DataFrame `df` is correct, but you are redefining it using the same data dictionary. You can directly use `data` to create the DataFrame: `df = pd.DataFrame(data)`.